script/Arcueid/Day 8/08_00_ARC08_2.txt

```diff
@@ -454,6 +461,7 @@ With an uncharacteristically violent stride, Akiha takes her leave of the living
 [sha:4cfe4f9fc21abac386b549804013bfc59ec4cc8d]{
 -- Page 91, Offset 19895.
 -- 秋葉さまがおかわいそうです。恋人を連れてくるのでしたら、もっと普通のひとを連れて来てあげてくださいな。
+// What does the maid know
 I feel sorry for Akiha-sama.
  If you're going to invite a lover over,
  at least bring one that's a little more normal.
```

Comment:
The funny maid is the person who dragged his sorry ass from Arc's apartment, after all.